### ISSUE TITLE
Steam corrupting the filesystem

### ISSUE THREAD

**Issue by Botspot** [Botspot/pi-apps#128]
Several people have reported that the Steam app corrupts the sd card and prevents other commands from being run.  They've also reported that it renders the sd card unbootable.
I've been able to repeat the error.

Strangely, Steam worked flawlessly just a few days ago. What happened? I don't know.

This is top on the to-do list, and as it's a severe bug, I've [pushed an emergency announcement](https://github.com/Botspot/pi-apps-announcements/commit/032f95e824f3e2b0bdbe8bee7bfd1765a1e2f58f) to the [pi-apps-announcements repo](https://github.com/Botspot/pi-apps-announcements/commit/032f95e824f3e2b0bdbe8bee7bfd1765a1e2f58f).

Edit: it's fixed now. See: https://github.com/Botspot/pi-apps/issues/128#issuecomment-741512048

**Comment by Botspot** [comment]
I've narrowed this down to the steam.deb extraction step. This is exceedingly strange for two reasons:

1. Tom's Hardware has a [Steam Box86 installation tutorial, and it uses the same command](https://www.tomshardware.com/how-to/raspberry-pi-install-steam).  
![2020-12-08-185204_1920x1080_scrot](https://user-images.githubusercontent.com/54716352/101559414-b3ceaf80-3986-11eb-9f08-54b6560483f7.png)
2. This worked fine during testing, though not anymore.


Full output:
```
pi@raspberrypi:~ $ ar x steam.deb
pi@raspberrypi:~ $ sudo tar xvf data.tar.xz -C /
./
./usr/
./usr/bin/
./usr/share/
./usr/share/pixmaps/
./usr/share/pixmaps/steam.png
./usr/share/pixmaps/steam_tray_mono.png
./usr/share/doc/
./usr/share/doc/steam/
./usr/share/doc/steam/steam_subscriber_agreement.txt.gz
./usr/share/doc/steam/README
./usr/share/doc/steam-launcher/
./usr/share/doc/steam-launcher/changelog.all.gz
./usr/share/doc/steam-launcher/copyright
./usr/share/applications/
./usr/share/man/
./usr/share/man/man6/
./usr/share/man/man6/steam.6.gz
./usr/share/icons/
./usr/share/icons/hicolor/
./usr/share/icons/hicolor/256x256/
./usr/share/icons/hicolor/256x256/apps/
./usr/share/icons/hicolor/256x256/apps/steam.png
./usr/share/icons/hicolor/48x48/
./usr/share/icons/hicolor/48x48/apps/
./usr/share/icons/hicolor/48x48/apps/steam.png
./usr/share/icons/hicolor/24x24/
./usr/share/icons/hicolor/24x24/apps/
./usr/share/icons/hicolor/24x24/apps/steam.png
./usr/share/icons/hicolor/16x16/
./usr/share/icons/hicolor/16x16/apps/
./usr/share/icons/hicolor/16x16/apps/steam.png
./usr/share/icons/hicolor/32x32/
./usr/share/icons/hicolor/32x32/apps/
./usr/share/icons/hicolor/32x32/apps/steam.png
./usr/lib/
./usr/lib/steam/
./usr/lib/steam/bootstraplinux_ubuntu12_32.tar.xz
./usr/lib/steam/steam.desktop
./usr/lib/steam/bin_steam.sh
./usr/lib/steam/bin_steamdeps.py
./etc/
./etc/apt/
./etc/apt/trusted.gpg.d/
./etc/apt/trusted.gpg.d/steam.gpg
./etc/apt/sources.list.d/
./etc/apt/sources.list.d/steam.list
./lib/
./lib/udev/
./lib/udev/rules.d/
./lib/udev/rules.d/60-steam-vr.rules
./lib/udev/rules.d/60-steam-input.rules
./usr/bin/steamdeps
./usr/bin/steam
./usr/share/applications/steam.desktop
pi@raspberrypi:~ $ sudo
bash: /usr/bin/sudo: No such file or directory
```

**Comment by Botspot** [comment]
It appears that you can manually install the deb file with `dpkg`, even though it's an x86 deb. More testing is necessary, but we'll probably end up using this:
```
sudo dpkg -i /home/pi/steam.deb
```

**Comment by Botspot** [comment]
Okay I think I've worked out all the problems pertaining to the Seam app messing up your SD card with commit b74bf02ec4364bd848edbd5cb5a38eff5bf5c2aa.

So instead of copying files to `/`, now it installs the steam.deb properly using `dpkg`.  
I still have no explanation how extracting files could corrupt an OS to make it unbootable, but that's somehow what happened.  
As to why this corruption didn't occur a few days ago, I still have no explanation. Perhaps the steam.deb file was updated on their end?

Anyway, it should work now.
